crate2nix/Cargo.nix

```diff
@@ -104,6 +104,7 @@ rec {
         };
         features = {
         };
+        resolvedDefaultFeatures = [ ];
```

Comment:
I'd prefer leaving these out...

Reply:
This is actually necessary for the Rust changes to work, because those changes introduce a check for features in the resolved features list